xilem_core/src/view_sequences/mod.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+mod impl_array;
+mod impl_option;
+mod impl_tuples;
+mod impl_vec;
+mod without_elements;
```

Comment:
I'm not a huge fan of these modules being in a completely random part of the module tree from the trait that they're implementing.

Why is this not a set of modules beneath `view_sequence.rs`?

Reply:
Overall I think it matches a pattern we've been applying elsewhere: declare a trait at the root (or for masonry, in a separate crate) and then have a folder with nothing but implementations of that trait.